### PR TITLE
tests: Add benchmark tests for some GraphQL queries

### PR DESCRIPTION
**Pull request by rhybrillou**
### Description

See the title

### User-facing documentation

(*must be* 2 items and both *must be* checked)

- [x] CHANGELOG update is not needed
- [x] Documentation is not needed

### Testing

- [ ] inspected CI results

#### Automated testing

(*must be* at least 1 item and all items *must be* checked)

- [x] added unit tests
- [x] modified existing tests
  

#### How I validated my change

Manual run of `ROX_SCALE_TEST=true go test -v -tags sql_integration -bench . -timeout 7200s` in:
- `central/graphql/handler`
- `central/deployment/service`
- `central/image/service`
- `central/vulnmgmt/service`

### CODE REVIEW

File: central/graphql/handler/handler_benchmark_test.go [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+		scan {
+			operatingSystem
+			scanTime
+		}
+	}
+}`
+)
+
+func BenchmarkDeploymentExport(b *testing.B) {
+	_ = b
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:85]: ?

File: central/graphql/handler/handler_benchmark_test.go [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+	testHelper := &testutils.ExportServicePostgresTestHelper{}
+	err := testHelper.SetupTest(b)
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:87]: `require.NoError(b, testHelper.SetupTest(b))`

File: central/graphql/handler/handler_benchmark_test.go [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+	if err != nil {
+		b.Error(err)
+	}
+	defer testHelper.TearDownTest(b)
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:91]: I'm not sure if this is better
```suggestion
	b.Cleanup(func() { testHelper.TearDownTest(b) })
```

File: central/graphql/handler/handler_benchmark_test.go [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+				jsonBytes, err := json.Marshal(vals)
+				if err != nil {
+					ib.Error(err)
+				}
+				ib.ResetTimer()
+				for i := 0; i < ib.N; i++ {
+					ib.StopTimer()
+					server, err := getGraphQLServer(testHelper)
+					if err != nil {
+						ib.Error(err)
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:200]: ditto

File: central/graphql/handler/handler_benchmark_test.go [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+					}
+					ib.StartTimer()
+					client := server.Client()
+					requestData := bytes.NewBuffer(jsonBytes)
+					req, reqErr := http.NewRequest(http.MethodPost, server.URL, requestData)
+					if reqErr != nil {
+						ib.Error(reqErr)
+					}
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:208]: This should be before timer starts

File: central/graphql/handler/handler_benchmark_test.go [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+					resp, callErr := client.Do(req)
+					if callErr != nil {
+						ib.Error(callErr)
+					}
+					_ = resp.Body.Close()
+					if resp.StatusCode != http.StatusOK {
```
Comment (janisz) [inline central/graphql/handler/handler_benchmark_test.go:214]: `assert.Equal`

File: central/testutils/export.go [hunk 7]
```diff
@@ -199,6 +199,20 @@ func (h *ExportServicePostgresTestHelper) InjectDeployments(
 				IsClusterLocal: false,
 			}
 			container.Image = containerImage
+			for _, v := range container.Volumes {
```
Comment (janisz) [inline central/testutils/export.go:202]: Please add a comment why we need to set enums

File: central/graphql/handler/handler_benchmark_test.go (hunk 2 above)
Comment (rhybrillou) [reply central/graphql/handler/handler_benchmark_test.go:87]: Sounds good

**Pull request merged**